src/providers/flagsView.ts

```diff
@@ -46,7 +46,7 @@ export class LaunchDarklyTreeViewProvider implements vscode.TreeDataProvider<Fla
 			return Promise.resolve(element.children);
 		} else {
 			return Promise.resolve(
-				this.flagValues.map(function(flag) {
+				this.flagValues.map(function (flag) {
 					return flag;
 				}),
 			);
```

Comment:
Not too familiar with what `getChildren` should do, but in the case where `element` is undefined, it looks like we're just returning the `flagValues` unmodified - is that expected? If so, we could probably do something like this:
suggestion:
```ts
return Promise.resolve(element ? element.children : this.flagValues);
```